src/test/rgw/test_rgw_crypto.cc

```diff
@@ -33,7 +33,7 @@ class ut_get_sink : public RGWGetObj_Filter {
   ut_get_sink() {}
   virtual ~ut_get_sink() {}
 
-  int handle_data(bufferlist& bl, off_t bl_ofs, off_t bl_len) override
+  int handle_data(bufferlist& bl, off_t bl_ofs, off_t bl_len, const jaeger_tracing::jspan* const parent_span = nullptr) override
```

Comment:
as the classes of rgw is very interelated through inheritance, defining the virtual functions, etc so I had to give them these parameters otherwise my code wont compile

Reply:
try to minimize these. are you sure that "handle_data()" needs tracing?
please see this comment: https://github.com/ceph/ceph/pull/36878#issuecomment-698271375